apps/kg/dataloader/sampler.py

```diff
@@ -8,15 +8,15 @@
 import pickle
 import time
 
-# This partitions a list of edges based on relations to make sure
-# each partition has roughly the same number of edges and relations.
-def RelationPartition(edges, n):
+# This partitions a list of edges based on relations and make sure
+# each relation only fall into one partition.
+def StrictRelationPartition(edges, n):
```

Comment:
Have KG set up CI properly? This function name does not comply with the snake case convention.